packages/financial-templates-lib/src/price-feed/CryptoWatchPriceFeed.js

```diff
@@ -139,15 +147,30 @@ class CryptoWatchPriceFeed extends PriceFeedInterface {
       this.apiKey ? `&apiKey=${this.apiKey}` : ""
     ].join("");
 
-    const ohlcResponse = await this.networker.getJson(ohlcUrl);
-    if (!ohlcResponse || !ohlcResponse.result || !ohlcResponse.result[this.ohlcPeriod]) {
-      this.logger.error({
-        at: "CryptoWatchPriceFeed",
-        message: "Could not parse ohlc result🚨",
-        error: new Error(JSON.stringify(ohlcResponse))
-      });
-      return;
-    }
+    // 2. Send requests.
+    const [ohlcResponse, priceResponse] = await Promise.all([
```

Comment:
This will throw if `networker.getJson()` throws right? possibly due to server error

Reply:
Yeah, the internal promise will reject, which will cause the Promise.all to reject, which _should_ cause the await to throw.